build/apis/contract-chain-c-chain-api.md

```diff
@@ -41,7 +41,459 @@ where `blockchainID` is the ID of the blockchain running the EVM.
+* If `startIndex` is omitted, will fetch all UTXOs up to `limit`.
+* When using pagination \(ie when `startIndex` is provided\), UTXOs are not guaranteed to be unique across multiple calls. That is, a UTXO may appear in the result of the first call, and then again in the second call.
+* When using pagination, consistency is not guaranteed across multiple calls. That is, the UTXO set of the addresses may have changed between calls.
+* `encoding` sets the format for the returned UTXOs. Can be either “cb58” or “hex”. Defaults to “cb58”.
+
+#### **Example**
+
+Suppose we want all UTXOs that reference at least one of `C-avax1yzt57wd8me6xmy3t42lz8m5lg6yruy79m6whsf`.
+
+```text
+curl -X POST --data '{
+    "jsonrpc":"2.0",
+    "id"     :1,
+    "method" :"avax.getUTXOs",
+    "params" :{
+        "addresses":["C-avax1yzt57wd8me6xmy3t42lz8m5lg6yruy79m6whsf"],
+        "sourceChain": "X",
+        "limit": 3,
+        "startIndex": {
+            "address": "C-avax1yzt57wd8me6xmy3t42lz8m5lg6yruy79m6whsf",
+            "utxo": "22RXW7SWjBrrxu2vzDkd8uza7fuEmNpgbj58CxBob9UbP37HSB"
+        },
+        "encoding": "cb58"
+    }
+}' -H 'content-type:application/json;' 127.0.0.1:9650/ext/bc/C/avax
+```
+
+This gives response:
+
+```text
+{
+    "jsonrpc": "2.0",
+    "result": {
+        "numFetched": "3",
+        "utxos": [
+            "11QEQTor9xZ1TyCyq8aFVShdP7YjM1ug9KuPUuMpgvQVz5qjEzo244NbJomjciNUPqUr1cD455dXhVrVNopnMXTQrTFY5kqrEVAQ3Ng9AnapQrYVEYiWc32F5CQuD3N5sB1EhQmMdJr5pis1QLjMmRQmut7Maafwup1vEU",
+            "11Eo6c9iUz3ERtmHbdUb3nzzMaqFffFQStshEsSTiFQP5xqfmeaeCFHCBajmoJUdQRHtkChGAmPucDfuCyBAEyGmmv2w8b7dX5sATxV7HxHZE4eak14GMGVEr7v3ij1B8mE82cymTJJz1X3PpRk2pTaxwEnLWfh1aAiTFC",
+            "118mpEHsia5sYYvKUx4j56mA7i1yvmLNyynm7LcmehcJJwMVY65smT4kGQgyc9DULwuaLTrUcsqbQutCdajoJXBdPVqvHMkYBTYQKs7WSmTXH8v7iUVqZfphMnS7VxVjGU1zykeTnbuAoZt4cFMUJzd8JaZk5eC82zmLmT"
+        ],
+        "endIndex": {
+            "address": "C-avax1yzt57wd8me6xmy3t42lz8m5lg6yruy79m6whsf",
+            "utxo": "27q6nsuvtyT4mvXVnQQAXw1YKoTxCow5Qm91GZ678TU1SvUiC2"
+        },
+        "encoding": "cb58"
+    },
+    "id": 1
+}
+```
+
+### avax.import
+
+Finalize the transfer of a non-AVAX from the X-Chain to the C-Chain.
+Before this method is called, you must call the X-Chain's [`export`](./avm.md#avmexport) method to initiate the transfer.
+
+#### Signature
+
+```go
+avax.import({
+    to: string,
+    sourceChain: string,
+    assetID: string,
+    username: string,
+    password:string,
+}) -> {txID: string}
+```
+
+##### Request
+
+* `to` is the address the AVAX is sent to. This must be the same as the `to` argument in the corresponding call to the C-Chain's `export`.
+* `sourceChain` is the ID or alias of the chain the AVAX is being imported from. To import funds from the X-Chain, use `"X"`.
+* `assetID` is the assetID of the non-AVAX asset.
+* `username` is the user that controls `to`.
+
+##### Response
+
+* `txID` is the txid of the completed ImportTx.
```

Comment:
`txid` --> ID

Reply:
This appears to return `txID` to me

```zsh
curl --location --request POST 'http://localhost:9650/ext/bc/C/avax' \
--header 'Content-Type: application/json' \
--data-raw '{
    "method": "avax.import",
    "params": {
        "username":"",
        "password":"",
        "sourceChain": "X",
        "to":"0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC"
    },
    "jsonrpc": "2.0",
    "id": 1
}'

{
    "jsonrpc": "2.0",
    "result": {
        "txID": "2Mh25No4uJoECQBjsKRHsx75hRbVhD4Y26UGV5XcVPafQG3kwL"
    },
    "id": 1
}
```

Reply:
Sorry, should have been more clear. I meant in the description: 

"`txID` is the ID of the completed ImportTx.`